src/api/app/models/review.rb

```diff
@@ -94,23 +97,31 @@ def check_initial
       errors.add(:unknown, 'no reviewer defined')
     end
 
-    if by_user && !User.find_by_login(by_user)
+    if by_user
```

Comment:
I will change this as it will increase the readability. Let's talk about moving this to a dedicated Validator class in another PR.